core/templates/dev/head/services/PlaythroughIssuesService.js

```diff
@@ -58,7 +58,7 @@ oppia.factory('PlaythroughIssuesService', [
     var renderMultipleIncorrectIssueSuggestions = function(stateName) {
       var suggestions = [$sce.trustAsHtml(
```

Comment:
Oh dear, this is not so good. Is it possible to render in HTML instead? We should not be trusting stuff like this as HTML.

Reply:
Hmm, possibly? I'm not sure what you mean though — do you mean storing this as HTML in a mini-directive or something?

Reply:
Done.